aspnetcore/fundamentals/minimal-apis/includes/parameter-binding7.md

```diff
@@ -10,7 +10,7 @@ Supported binding sources:
 * Services provided by dependency injection
 * Custom
 
-Binding from form values is ***not*** natively supported in .NET.
+Binding from form values is ***not*** natively supported in .NET 6 and 7.
```

Comment:
Is it worth saying, it's available in 8+ and append ?`view=aspnetcore-8.0&preserve-view=true` to the link?

Probably not worth it as 8 will be out soon and is LTS and the default version.

Reply:
Are you taking care of this in What's new in 8?

Reply:
> Probably not worth it as 8 will be out soon and is LTS and the default version.

Agreed - people looking at the old version 7 of the doc will know to check out version 8.

Reply:
> Are you taking care of this in What's new in 8?

Yes